Backend_Development_With_PHP/readme.md

```diff
@@ -0,0 +1,97 @@
+# Girl Script Winter Of Contribution
+
+<img src="https://github.com/girlscript/winter-of-contributing/blob/main/banner_readme.png"  style="max-width: 100%; height: auto;"/>
+
+# Topics in Backend Development With PHP
+
+	- Links are not working right now but we will update it soon :)
+
+### 1.&nbsp;&nbsp; <a href="#">Getting started</a>
+  - <a href="#">Use the Current Stable Version (8.0)</a>
+  - <a href="#">Built-in Web Server</a>
+  - <a href="#">Mac Setup</a>
+  - <a href="#">Ubuntu / Linux Setup</a>
+  - <a href="#">Windows Setup</a>
+  - <a href="#">Common Directory Structure</a>
+
+### 2.&nbsp;&nbsp; <a href="#">Code style guide</a>
+
+### 3.&nbsp;&nbsp; <a href="#">Language Highlights</a>
+
+  - <a href="#">Programming Paradigms</a>
+  - <a href="#">Namespaces</a>
+  - <a href="#">Standard PHP Library</a>
+  - <a href="#">Command Line Interface</a>
+  - <a href="#">Xdebug</a>
+
+### 4.&nbsp;&nbsp; <a href="#">Dependency Management</a>
+
+  - <a href="#">Composer and Packagist</a>
+  - <a href="#">PEAR</a>
+
+### 5.&nbsp;&nbsp; <a href="#">Coding Practices<a>
+
+  - <a href="#">The Basics</a>
+  - <a href="#">Date and Time</a>
+  - <a href="#">Design Patterns</a>
+  - <a href="#">Working with UTF-8</a>
+  - <a href="#">Internationalization and Localization</a>
+
+### 6.&nbsp;&nbsp; <a href="#">Dependency Injection</a>
+
+  - <a href="#">Basic Concept</a>
+  - <a href="#">Complex Problem</a>
+  - <a href="#">Containers</a>
+
+### 7.&nbsp;&nbsp; <a href="#">Databases</a>
+
+  - <a href="#">MySQL Extension</a>
+  - <a href="#">PDO Extension</a>
+  - <a href="#">Interacting with Databases</a>
+  - <a href="#">Abstraction Layers</a>
+
+### 8.&nbsp;&nbsp; <a href="#">Templating</a>
+
+  - <a href="#">Benefits</a>
+  - <a href="#">Plain PHP Templates</a>
+  - <a href="#">Compiled Templates</a>
+
+
+### 9.&nbsp;&nbsp; <a href="#">Errors and Exceptions</a>
+
+  - <a href="#">Error Handling</a>
+  - <a href="#">Exceptions Handling</a>
+
+
+### 10.&nbsp;&nbsp; <a href="#">Security</a>
+
+  - <a href="#">Web Application Security</a>
+  - <a href="#">Password Hashing</a>
+  - <a href="#">Data Filtering</a>
+  - <a href="#">Configuration Files</a>
+  - <a href="#">Register Globals</a>
+  - <a href="#">Error Reporting</a>
+
+
+### 11.&nbsp;&nbsp; <a href="#">Servers and Deployment</a>
+
+  - <a href="#">Platform as a Service (PaaS)</a>
+  - <a href="#">Virtual or Dedicated Servers</a>
+  - <a href="#">Shared Servers</a>
+  - <a href="#">Building Your Application</a>
+
+
+### 12.&nbsp;&nbsp; <a href="#">Virtualization</a>
+
+  - <a href="#">Vagrant</a>
+  - <a href="#">Docker</a>
+
+
+### 13.&nbsp;&nbsp; <a href="#">Caching</a>
+
+  - <a href="#">Opcode Cache</a>
+  - <a href="#">Object Caching</a>
+
+### 14.&nbsp;&nbsp; <a href="#">Documenting your Code</a>
+
+  - <a href="#">PHPDoc</a>
```

Comment:
Couple of things.

1. Why do we have a lot of `&nbsp;&nbsp;`?
2. Use Markdown for everything, don't use HTML tags.
3. All the spacing needs to be right. Some have one line, two line and zero line spaces.
4. Remove trailing spaces on the top and bottom.